ops/solana/solana.go

```diff
@@ -364,98 +415,100 @@ func (d Deployer) InitOCR(keys []opsChainlink.NodeKeys) error {
+		d.Account[OCRFeed],
+	); err != nil {
+		return errors.Wrap(err, "proposing OCR2 offchain config failed")
 	}
 
+	fmt.Println("Proposing Payees...")
 	input = map[string]interface{}{
-		"oracles": oracles,
-		"f":       threshold,
+		"operators":          oracles,
+		"proposalId":         d.Account[Proposal],
+		"allowFundRecipient": true,
 	}
 
 	jsonInput, err = json.Marshal(input)
 	if err != nil {
 		return err
 	}
 
-	fmt.Println("Setting config...")
-	err = d.gauntlet.ExecCommand(
-		"ocr2:set_config",
+	if err = d.gauntlet.ExecCommand(
+		"ocr2:propose_payees",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("state", d.Account[OCRFeed]),
+		d.gauntlet.Flag("link", d.Account[LINK]),
+		d.gauntlet.Flag("proposalId", d.Account[Proposal]),
 		d.gauntlet.Flag("input", string(jsonInput)),
-	)
-
-	if err != nil {
-		return errors.Wrap(err, "setting OCR 2 config failed")
+		d.Account[OCRFeed],
+	); err != nil {
+		return errors.Wrap(err, "setting OCR 2 payees failed")
 	}
 
 	input = map[string]interface{}{
-		"operators": operators,
+		"observationPaymentGjuels":  1,
+		"transmissionPaymentGjuels": 1,
 	}
 
 	jsonInput, err = json.Marshal(input)
 	if err != nil {
 		return err
 	}
 
-	fmt.Println("Setting Payees...")
-	err = d.gauntlet.ExecCommand(
-		"ocr2:set_payees",
+	fmt.Println("Finalize proposal...")
+	if err = d.gauntlet.ExecCommand(
+		"ocr2:finalize_proposal",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("state", d.Account[OCRFeed]),
-		d.gauntlet.Flag("link", d.Account[LINK]),
-		d.gauntlet.Flag("TESTING_ONLY_IGNORE_PAYEE_VALIDATION", "true"),
-		d.gauntlet.Flag("input", string(jsonInput)),
-	)
-
-	if err != nil {
-		return errors.Wrap(err, "setting OCR 2 payees failed")
+		d.gauntlet.Flag("proposalId", d.Account[Proposal]),
+	); err != nil {
+		return errors.Wrap(err, "committing OCR 2 set offchain config failed")
 	}
 
+	fmt.Println("Accept proposal...")
 	input = map[string]interface{}{
-		"observationPaymentGjuels":  1,
-		"transmissionPaymentGjuels": 1,
+		"version":  2,
+		"f": threshold,
+		"oracles": oracles,
+		"offchainConfig": offchainConfig,
+		"randomSecret": testingSecret,
 	}
 
 	jsonInput, err = json.Marshal(input)
 	if err != nil {
 		return err
 	}
+	if err = d.gauntlet.ExecCommand(
+		"ocr2:accept_proposal",
+		d.gauntlet.Flag("network", d.network),
+		d.gauntlet.Flag("proposalId", d.Account[Proposal]),
+		d.gauntlet.Flag("secret", testingSecret),
+		d.gauntlet.Flag("input", string(jsonInput)),
+		d.Account[OCRFeed],
+	); err != nil {
+		return errors.Wrap(err, "failed to accept proposal")
+	}
 
 	fmt.Println("Setting Billing...")
-	err = d.gauntlet.ExecCommand(
+	if err = d.gauntlet.ExecCommand(
 		"ocr2:set_billing",
 		d.gauntlet.Flag("network", d.network),
-		d.gauntlet.Flag("state", d.Account[OCRFeed]),
 		d.gauntlet.Flag("input", string(jsonInput)),
-	)
-
-	if err != nil {
+		d.Account[OCRFeed],
+	); err != nil {
 		return errors.Wrap(err, "setting OCR 2 billing failed")
```

Comment:
```suggestion
		return errors.Wrap(err, "'ocr2:set_billing' call failed")
```

If this is surfaced through an original error, maybe even is unnecessary to wrap.